src/user/authorization/SignUpPage.tsx

```diff
@@ -0,0 +1,86 @@
+    const toastIdRef = React.useRef()
+
+    const handleClick = () => setShow(!show)
+  
+    const [username, setUsername] = React.useState("")
+    const [password, setPassword] = React.useState("")
+
+    const handleSubmit = () => {
+        let creds: Creds
+        creds = {username: username, password: password}
+    
+        authorizationClientInstance.signUp(creds).then(
+            res => {
+                console.log(res)
+                if(res.status === 201) {
+                    showCreatedAccount()
+                }
+            }
+
+        ).catch(error => {
+            console.log(error)
+            showSigningUpFailure()
+        });
+    }
+
+    const showCreatedAccount = () => {
+        toastIdRef.current = toast({description: "Created account successfully!", status: "success"}) as any
+    }
+
+    const showSigningUpFailure = () => {
+        toastIdRef.current = toast({description: "Error!", status: "error"}) as any
+    }
+  
+    return (
+        <>
+            <Box className="AuthorizationPageBox" rounded="lg">
+                <Box className="AuthorizationPageBoxContent">
+                    <Heading as="h3" size="lg" paddingTop="20px" marginBottom="5px">Sign up</Heading>
+                    <Text fontSize="2xl" marginBottom="25px" className="Logo">service logo</Text>
+                        <FormControl isRequired mt={6}>
+                            <FormLabel textAlign="left" mb="8px"> Username or email</FormLabel>
+                                <Input
+                                    className="Center"
+                                    pr="4.5rem"
+                                    bgColor="white"
+                                    onChange={event => setUsername(event.currentTarget.value)}
+                                />
+                        </FormControl>
+
+                        <FormControl isRequired mt={6}>
+                            <FormLabel textAlign="left" mb="8px"> Password</FormLabel>
+                            <InputGroup className="Center" size="md" > 
+                                <Input
+                                    pr="4.5rem"
+                                    type={show ? "text" : "password"}
+                                    bgColor="white"
+                                    onChange={event => setPassword(event.currentTarget.value)}
+                                />
+                        
+                                <InputRightElement width="4.5rem">
+                                    <Button h="1.75rem" size="sm" onClick={handleClick}>
+                                    {show ? "Hide" : "Show"}
+                                    </Button>
+                                </InputRightElement>
+                            </InputGroup>
+                        </FormControl>
+                        <Button  onClick={handleSubmit} colorScheme="green" marginTop="20px" size="sm">Sign up</Button>
```

Comment:
Prettier or other formatting would be nice

```suggestion
                        <Button onClick={handleSubmit} colorScheme="green" marginTop="20px" size="sm">Sign up</Button>
```